docs/source/ecosystem.rst

```diff
@@ -69,6 +69,7 @@ There are many different implementations of the Dask distributed cluster.
   installations.
 - `dask-cloudprovider <https://cloudprovider.dask.org>`_: Deploy Dask on various cloud platforms such as AWS, Azure, and GCP
   leveraging cloud native APIs.
+- `Coiled <https://www.coiled.io/>`_: Deploy Dask in the cloud from your AWS or Google Cloud accounts.
```

Comment:
I think that we want to call this out as a managed/commercial offering in this description. That's the first major distinction from Dask cloud provider

Reply:
Sounds good, I've added a `Commercial Dask Deployment Options` section at the bottom with Coiled listed as one option.